client/lib/directly/test/index.js

```diff
@@ -96,22 +80,20 @@ describe( 'index', () => {
 		const name = 'Richie Rich';
 		const email = 'richie@richenterprises.biz';
 
-		it( 'does nothing if Directly hasn\'t been initialized', () => {
+		it( 'initializes Directly if it hasn\'t already been initialized', () => {
```

Comment:
Hm, using backticks gives a lint warning:

```
Warning: Strings must use singlequote.
```

Which is better — lint warning or escaping a quote? :)